src/ray/core_worker/transport/direct_actor_transport.h

```diff
@@ -420,9 +420,10 @@ class SchedulingQueue {
 
 class CoreWorkerDirectTaskReceiver {
  public:
-  using TaskHandler = std::function<Status(
-      const TaskSpecification &task_spec, const ResourceMappingType &resource_ids,
-      std::vector<std::shared_ptr<RayObject>> *return_objects)>;
+  using TaskHandler =
+      std::function<Status(const TaskSpecification &task_spec,
+                           const std::shared_ptr<ResourceMappingType> resource_ids,
+                           std::vector<std::shared_ptr<RayObject>> *return_objects)>;
```

Comment:
Could you add a comemnt here that resources_ids is only used for non-actor tasks and will be empty for actor tasks?

Reply:
Done